src/metabase/pulse/render/body.clj

```diff
@@ -193,15 +203,18 @@
   [_ _ timezone-id :- (s/maybe s/Str) card {:keys [cols] :as data}]
   (let [[x-axis-rowfn y-axis-rowfn] (common/graphing-column-row-fns card data)
         rows                        (common/non-nil-rows x-axis-rowfn y-axis-rowfn (:rows data))
-        max-value                   (apply max (map y-axis-rowfn rows))]
+        row-values                  (map y-axis-rowfn rows)
+        min-value                   (min 0 (apply min row-values))
```

Comment:
I'm not sure what the Best Approach is here. This imitates the old way, which would make values of `100, 105, 110` look really close since the baseline is zero.

An alternate way would be to make the baseline the true min value, so `100` looks like nothing and `110` looks huge.

Both have pros and cons, so it seemed best to just leave it as-is.